### ISSUE TITLE
just 2 questions, thanks

### ISSUE THREAD

**Issue by BenJaminB1ue** [Sohojoe/ActiveRagdollStyleTransfer#13]
Hi, sohojoe, it's a great project!
Could you just tell me why you said your project StyleTransfer001 was fail? In other words, could you tell me what's the main difference between 001 and 002 project?
By the way, is the function "use random index" in StyleTransfer001 for training rather than for inferencing? But how does it works in 002 project because I can't find such function available.
It will be so kind of you to answer me, thanks.

**Comment by Sohojoe** [comment]
I was able to get a basic walk cycle working with 001 but struggled with more complex (i don't think I got run working and backflip took months on 002) - the main reason for moving to 002 was because I had more experience with the humanoid figure and knew I should be able to get it to run whereas with the figure I used in 001, it was not clear if it was not working because of a mistake in my code or a limitation of the figure. Note: I fixed the readme and you can see an animation of 001 walking - [here](https://github.com/Sohojoe/ActiveRagdollStyleTransfer/blob/master/README.md)

The idea of 'RandomIndex' was to randomize the start animation frame of each training episode. I didn't find this helped and have to continued to mess around with it.

Note: I'm no longer maintaining this codebase as I've implemented the StyleTransfer code into the main [MarathonEnvs](https://github.com/Unity-Technologies/marathon-envs) repro - I'm still ironing out a few kinks but that code is more up to date and robust

**Comment by BenJaminB1ue** [comment]
I'm very glad to hear from you, Sohojoe, and the answer is what I really expected, thanks!
By the way, during the period I tested the StyleTransfer001 project, the animator I downloaded from mixamo.com can play sucessfully in Unity editor, but can NOT play in .exe built from Unity editor, the character just get stuck, will you have any idea of the problem?
In addittion, would you give me any suggestion about how to train agents with increasing reward curves( such as change hyper parameters in config.yaml), for it may not be plausible that agents work much worse after training more.
![image](https://user-images.githubusercontent.com/44258961/82154391-08ea8d00-98a0-11ea-896b-8ee417a9eb1b.png)

Best wishes.

**Comment by Sohojoe** [comment]
Hi @BenJaminB1ue, for the backflip i trained for 128m training steps - so you could try increasing the number of steps. Is the entropy chart decreasing? if not, you can decrease the beta. Another thing is that the learning rate (lr), if it is too high, then training will be less stable.

**Comment by Sohojoe** [comment]
regarding not playing in the .exe, I've not seen this before and not sure i tested with v001

**Comment by BenJaminB1ue** [comment]
I gonna have a try, thank you so much!
